core/src/main/java/io/kestra/core/models/triggers/types/Schedule.java

```diff
@@ -283,7 +292,7 @@ public Optional<Execution> evaluate(ConditionContext conditionContext, TriggerCo
             .build();
 
         // add inputs and inject defaults
-        if (inputs.size() > 0) {
+        if (!inputs.isEmpty()) {
```

Comment:
if the users don't have any inputs on the schedule, it must works also.
please add a unit test with no inputs at all and only defaults

Reply:
This is the test I added: no trigger input but a flow input with a default.
I'll add a test with a trigger input.